datasets/Womens_Clothing_E-Commerce_Reviews/train_config.json

```diff
@@ -5,15 +5,21 @@
   "combine_feat_method": "text_only",
   "experiment_name": "Unimodal Bert Base Uncased",
   "model_name_or_path": "bert-base-uncased",
-  "gpu_num": 0,
   "do_train": true,
+  "categorical_encode_type": "binary",
+  "numerical_transformer_method": "quantile_normal",
   "tokenizer_name": "bert-base-uncased",
-  "per_device_train_batch_size": 12,
+  "use_simple_classifier": false,
+  "logging_dir": "./logs_clothing_review/bertbase_gating_on_cat_and_num_feats_then_sum_full_model_lr_3e-3/",
```

Comment:
is it possible to avoid hardcoded paths like this?

Reply:
What would you suggest instead? These seem to be primarily intended to serve as examples for users more than anything else.